azext_iot/operations/digitaltwin.py

```diff
@@ -175,38 +175,55 @@ def iot_digitaltwin_property_update(cmd, interface_payload, device_id,
         raise CLIError(unpack_msrest_error(e))
 
 
-def iot_digitaltwin_monitor_events(cmd, device_id, interface, source_model, repo_endpoint=PNP_ENDPOINT, repo_id=None,
-                                   consumer_group='$Default', timeout=300, hub_name=None, resource_group_name=None,
-                                   yes=False, properties=None, repair=False, login=None, repo_login=None):
+def iot_digitaltwin_monitor_events(cmd, device_id=None, device_query=None, interface=None, source_model='public',
```

Comment:
I was wondering why source_model is not an enum type?

Reply:
Updated to enum